mysqlparse/grammar/alter_table.py

```diff
@@ -144,10 +144,14 @@
 # Source: http://dev.mysql.com/doc/refman/5.7/en/alter-table.html
 #
 
+_database_name = (Optional(Word(alphanums + "`_") + FollowedBy('.') +
+                           Suppress('.'), default=None)
+                  .setResultsName("database_name")
+                  .setParseAction(lambda toks: toks[0]))
```

Comment:
Good point @adamchainz , I somehow never used other chars, thus forgot to considered them. Will create a separate issue to add support.

@jgysland thanks for another contribution! Could you add a line under `PENDING` in `HISTORY.rst` in this change? Also, does this require a release for you? Reason being I would like to assemble a few of those (e.g. a basic support for `CREATE TABLE` #20) before I issue a `0.1.7`, but since there are no clear release guidelines here - I can be flexible if it actually helps you.